usersession/agent/rest_api.go

```diff
@@ -431,11 +433,43 @@ func postPendingRefreshNotification(c *Command, r *http.Request) Response {
 	return SyncResponse(nil)
 }
 
+func guessAppIcon(si *snap.Info) string {
```

Comment:
How will this behave for icons prefixed with `${SNAP}`? Can we have a test for this?

please check [rewriteIconLine](https://github.com/snapcore/snapd/blob/master/wrappers/desktop.go#L147) for reference on how `Icon` desktop entry is constructed

Reply:
my bad, please ignore this, ${SNAP} should already be expanded.